source/guides/network/wifi.rst

```diff
@@ -0,0 +1,99 @@
+.. _wi-fi:
+
+Configure Wi-Fi with NetworkManager
+###################################
+
+We recommend using `NetworkManager <https://developer.gnome.org/NetworkManager/stable/NetworkManager.html>`_ to manage network connections. If you choose to
+connect to Wi-Fi while using the live image, your Wi-Fi settings will be
+added to your system during the installation process.
+
+NetworkManager provides three simple methods for configuring Wi-Fi.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Desktop GUI (Graphical User Interface)
+**************************************
+
+1. Click anywhere on the icons at the right side of the top of the screen to
+   bring up a menu and click on :guilabel:`Wi-Fi Not Connected` then
+   choose :guilabel:`Select Network`.
+
+   .. figure:: /_figures/wifi/wifi-1.1.png
+
+2. Scroll through the list and select the network you'd like to connect to and
+   click :guilabel:`Connect`.
+
+   .. figure:: /_figures/wifi/wifi-2.png
+
+3. Enter the password and click :guilabel:`Connect`.
+
+   .. figure:: /_figures/wifi/wifi-3.png
+
+4. The Wi-Fi icon should now show the signal strength of the connection.
+
+   .. figure:: /_figures/wifi/wifi-4.png
+
+5. If you are installing using the live image, resume the
+   :ref:`installation process <install-on-target-end>` now. Your Wi-Fi
+   configuration will automatically be included in the install. 
+
+   .. figure:: /_figures/wifi/wifi-5.png
+
+CLI (Command Line Interface)
+****************************
+
+#. List the available Wi-Fi networks
+
+   .. code-block:: bash
+
+      nmcli device wifi list
+
+   .. code-block:: console
+
+      IN-USE  SSID                           MODE   CHAN  RATE        SIGNAL  BARS  SECURITY         
+              1st Network                    Infra  1     54 Mbit/s   65      ▂▄▆_  --               
+              2nd Network                    Infra  1     130 Mbit/s  52      ▂▄__  --               
+              3rd Network                    Infra  10    195 Mbit/s  29      ▂___  WPA2             
+
+#. Join the network.
+
+   .. code-block:: bash
+
+      nmcli device wifi connect $SSID password $password
```

Comment:
The problem with this is that it will try to pop up a GUI dialog, which would interfere with terminal only usage. Perhaps a note to use a script or the TUI, if this is a concern.